lms/djangoapps/teams/serializers.py

```diff
@@ -17,6 +18,10 @@ class UserMembershipSerializer(serializers.ModelSerializer):
 
     Used for listing team members.
     """
+    profile_configuration = deepcopy(settings.ACCOUNT_VISIBILITY_CONFIGURATION)
+    profile_configuration['shareable_fields'].append('url')
```

Comment:
We include it in general because an expanded object might itself have references that have not been expanded. If you want all that information in one shot you could ask to have them expanded, or else you can use the URL to make subsequent calls to fetch more details. In this case it doesn't buy us much as there are no additional references, but IMO it makes sense to keep it for consistency.